### PR TITLE
Add network policy in demo for dpdk

### PR DESCRIPTION
**Pull request by SchSeba**
This PR adds the requested yamls to configure the second interface in the demo environment to use vfio for dpdk.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: feature-configs/demo/sriov/dpdk-network.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+kind: SriovNetwork
+metadata:
+  name: dpdk-network
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |
+    {
+      "type": "dhcp"
+    }
+  networkNamespace: dpdk-testing
```
Comment (fedepaol) [inline feature-configs/demo/sriov/dpdk-network.yaml]: Do we need the creation of the namespace somewhere here?

File: feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames:
+      - eno2
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+    node-role.kubernetes.io/worker-cnf: ""
```
Comment (fedepaol) [inline feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml]: This selector can be one with `and` (or probably sticking to `worker-cnf` is fine)

File: feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: dpdk-network-node-policy-dpdk
```
Comment (fedepaol) [inline feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml]: Can this be either `dpdk-network-node-policy` or `network-node-policy-dpdk` ?

File: feature-configs/demo/sriov/dpdk-network.yaml (hunk 1 above)
Comment (SchSeba) [reply feature-configs/demo/sriov/dpdk-network.yaml]: in this file take a look below :)

File: feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml (hunk 3 above)
Comment (SchSeba) [reply feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml]: ops yep my bad nice catch

File: feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml (hunk 2 above)
Comment (SchSeba) [reply feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml]: sure

File: feature-configs/demo/sriov/dpdk-network.yaml (hunk 1 above)
Comment (fedepaol) [reply feature-configs/demo/sriov/dpdk-network.yaml]: Lol sorry I was looking for another file

**Pull request merged**